### PR TITLE
docs(samples): disable bucket lifecycle management

### PR DESCRIPTION
**Pull request by athakor**

### CODE REVIEW

**Comment by athakor** [comment]
@frankyn PTAL

File: google-cloud-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITBucketSnippets.java [hunk 1]
```diff
@@ -271,6 +272,12 @@ public void testEnableLifecycleManagement() {
     assertEquals(1, storage.get(BUCKET).getLifecycleRules().size());
   }
 
+  @Test
+  public void testDisableLifecycleManagement() {
+    DisableLifecycleManagement.disableLifecycleManagement(PROJECT_ID, BUCKET);
```
Comment (frankyn) [inline google-cloud-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITBucketSnippets.java:277]: please add a lifecycle policy to delete

Comment (athakor) [reply google-cloud-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITBucketSnippets.java:288]: done

**Comment by athakor** [comment]
@frankyn @jkwlui PTAL

**Pull request merged**